recipes/xcube/meta.yaml

```diff
@@ -0,0 +1,74 @@
+{% set name = "xcube" %}
+{% set version = "0.2.0" %}
+
+package:
+  name: "{{ name }}"
+  version: "{{ version }}"
+source:
+  url: "https://github.com/dcs4cop/{{ name }}/archive/v{{ version }}.tar.gz"
+  sha256: 35a934429ff173b383203395ad4ad513c405a15bd5a6c2a319d4d8fa6816f116
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+  entry_points:
+    - xcube = xcube.cli.cli:main
+    - xcube-gen = xcube.cli.gen:main
+    - xcube-restime = xcube.cli.restime:main
+requirements:
+  host:
+    # Python
+    - python >=3.6
+    - pip
+  run:
+    # Python
+    - python >=3.6
+    # Required
+    - affine >=2.2
+    - blas =*=mkl
```

Comment:
@isuruf is this the best way to use `mkl` only on conda-forge these days?

Reply:
Yes

Reply:
OK. Those three letters is all we need :smile: